tests/auth/registration/registration.spec.js

```diff
@@ -0,0 +1,170 @@
+import {expect, test} from '@playwright/test';
+
+test.describe('Registration', () => {
+
+    async function openRegistrationPopup(page) {
+        return await test.step('Open registration popup', async () => {
+            await page.goto('/')
+
+            const signInButton = page.locator('button.header_signin')
+            await expect(signInButton, 'Sign in button should be visible').toBeVisible()
+            await expect(signInButton, 'Sign in button should be enabled').toBeEnabled()
+
+            await signInButton.click()
+
+            const popup = page.locator('div.modal-dialog')
+
+            const loginPopup = popup.filter({has: page.locator('h4.modal-title', {hasText: 'Log in'})})
+            await expect(loginPopup, 'Sign in popup should be visible').toBeVisible()
+
+            const registrationButton = loginPopup.locator('.modal-footer .btn-link')
+            await registrationButton.click()
```

Comment:
use this button for opening sign up popup
![image](https://github.com/m-tetiana/aqa-playwright/assets/53534240/a3294b5f-68c4-44fe-9aa0-5afd82ff5ac5)